import_export/utils.py

```diff
@@ -23,3 +27,61 @@ def __enter__(self):
     def __exit__(self, *args):
         if self.using_transactions:
             self.context_manager.__exit__(*args)
+
+
+def diff_lines_to_words(text1, text2):
```

Comment:
Im thinking these functions should be put into another module, perhaps `diff.py`. I'm afraid that we are going to keep on putting stuff into `utils.py` as the kind of catch-all module.  What do you think?

Reply:
I agree